README.md

```diff
@@ -22,7 +22,7 @@ $ make images
  => => writing image sha257:96a86e6ebb38238569c007491c3e86a056340ceb9e4a3e66959bfa6a6ca8f8a0
  => => naming to quay.io/lvh-images/root-images
 $ c=$(docker create sha256:96a86e6ebb38238569c007491c3e86a056340ceb9e4a3e66959bfa6a6ca8f8a0)
-$ docker cp $c:/data/images/base.qcow2.zst /tmp
+$ docker cp $c:/data/images/kind_bpf-next.qcow2.zst /tmp
```

Comment:
I think that's not correct. The original example is about `root-images`, and kind_bpf-next exists in the `kind` images. For the `kind` and `complexity-test` images, it might make sense to have a symbolic link  always pointing to the proper file (since these only contain one image).

Reply:
For the README, specifically it might make sense to add an examples for kind and complexity-test images as well.

Reply:
Sorry, missed your comments. Sure, I'll extend the README with kind images examples.